src/management/GuardianManager.js

```diff
@@ -104,6 +104,48 @@ var GuardianManager = function(options) {
     guardianFactorsProvidersAuth0RestClient,
     options.retry
   );
+
+  /**
+   * Provides an abstraction layer for retrieving Guardian policies.
+   *
+   * @type {external:RestClient}
+   */
+  var guardianPoliciesAuth0RestClient = new Auth0RestClient(
+    options.baseUrl + '/guardian/policies',
+    clientOptions,
+    options.tokenProvider
+  );
+  this.policies = new RetryRestClient(guardianPoliciesAuth0RestClient, options.retry);
+
+  /**
+   * Provides an abstraction layer for retrieving Guardian phone factor selected provider.
+   *
+   * @type {external:RestClient}
+   */
+  var guardianFactorsPhoneSelectedProviderAuth0RestClient = new Auth0RestClient(
+    options.baseUrl + '/guardian/factors/sms/selected-provider',
```

Comment:
We discussed whether this should be `/sms/selected-provider` (the one that is already in production) or the new alias `/phone/selected-provider` (almost ready, but not available in PSaaS until July's release). To make the release smooth and ship this early, I opted to use the `/sms/selected-provider`. cc @joseluisdiaz

Reply:
👍

Reply:
ok, shall we create a task to change it in the future ?